smtp/init.lua

```diff
@@ -151,10 +151,10 @@ curl_mt = {
         --
         request = function(self, url, from, to, body, opts)
             opts = opts or {}
-            to = to or {}
-            if not body or not url or not from then
+            if not body or not url or not from or not (to or opts.cc or opts.bcc) then
```

Comment:
The error that is given in the case does not help to spot the usage problem. I would give its own error message in the case.

Reply:
Fixed